io/src/test/java/org/commonjava/maven/ext/io/rest/VersionTranslatorTest.java

```diff
@@ -176,19 +166,8 @@ public void testTranslateVersionsFailNoResponse()
     @Test( timeout = 2000 )
     public void testTranslateVersionsPerformance() throws RestException
     {
-        Logger logbackLogger = ( (Logger) LoggerFactory.getLogger( Logger.ROOT_LOGGER_NAME ) );
-        Level originalLevel = logbackLogger.getLevel();
-
-        try
-        {
-            // Disable logging for this test as impacts timing.
-            logbackLogger.setLevel( Level.OFF );
-            versionTranslator.translateVersions( aLotOfGavs );
-        }
-        finally
-        {
-            logbackLogger.setLevel( originalLevel );
-        }
+        // Disable logging for this test as impacts timing.
```

Comment:
Comment is still here,  but it looks like the logger code was removed.

Reply:
Great spot!